frontend/public/scripts/getContent.js

```diff
@@ -0,0 +1,50 @@
+// This function gets the content of the page requested
+
+export const getContent = (apiUrl) => {
+  const getCookieValue = (cookieName) => {
+    const cookies = document.cookie.split("; ");
+    for (const cookie of cookies) {
+      const [key, value] = cookie.split("=");
+      if (key === cookieName) {
+        return value;
+      }
+    }
+    return null;
+  };
+
+  const accessToken = getCookieValue("accessToken");
+
+  if (!accessToken) {
+    window.open("login.html", "_self");
+    return;
+  }
+  const verifyToken = async () => {
+    try {
+      const response = await fetch(apiUrl, {
+        headers: { Authorization: `Bearer ${accessToken}` },
+      });
+      if (!response.ok) {
+        window.open("login.html", "_self");
+        return;
+      } else {
+        const userRes = await fetch(
+          "http://localhost:4000/api/v1/users/currentUser",
```

Comment:
I don't think it is necessary to get the current user from the api? The current user sends requests using a token and this information should tell you who the user is.